src/flows/actions/flowsThunks.ts

```diff
@@ -0,0 +1,22 @@
+import {Dispatch} from 'redux'
+
+import {Notebook} from 'src/client/notebooksRoutes'
+
+import {getNotebooks as fetchNotebooks} from 'src/client/notebooksRoutes'
+import {setNotebooks} from 'src/flows/actions/flowsActions'
+
+export const getNotebooks =
+  (orgID: string) => async (dispatch: Dispatch<any>) => {
+    try {
+      const response = await fetchNotebooks({query: {orgID}})
+
+      if (response.status !== 200) {
+        throw new Error()
+      }
```

Comment:
I know this an example of "do as I say, not as I do," with respect to creating an API layer that manages the communciation between the API and the application. My thinking here is that this is code that is being written so that it can be disappeared and that we don't need to build this with long-term maintenance and consistency in mind